wrappers/IWearActuators/src/IWearActuatorsWrapper.cpp

```diff
@@ -59,6 +62,18 @@ bool IWearActuatorsWrapper::open(yarp::os::Searchable& config)
         return false;
     }
 
+    if (!config.check("rightHand") || !config.find("rightHand").isString()) {
+        yError() << LogPrefix << "rightHand parameter not found";
+        return false;
+    }
```

Comment:
This works only for the glove part, but this is used also for other devices. I don't think this can be used here

Reply:
Right. Then I will add a similar parameter to the wrapper part.